index.html

```diff
@@ -95,12 +95,12 @@ <h3>We are a team of students in GITAM that aim to promote coding as being fun,
 					</div>
 				</div>
 				<div class="intro-content col-sm-10 offset-sm-1 col-lg-5 offset-lg-1 d-flex justify-content-center align-items-center flex-column">
-					<h3>You just missed Code Hackathon Week #10 ☹️</h3>
+					<h3>💻 Sign up for Code Hackathon Week #12 📈</h3>
 					<br />
-					<img style="background: rgb(26, 18, 69);" class="poster old-pic" src="assets/poster/w10.jpg" alt="Code Hackathon Week #10 Poster">
+					<img style="background: rgb(26, 18, 69);" class="poster old-pic" src="assets/poster/w12.jpg" alt="Code Hackathon Week #12 Poster">
```

Comment:
The `old-pic` class is only used for inactive/completed event posters.

Reply:
Alright, I'll remove that class then.

Reply:
Done Annaya, please check.